templates/web/base/questionnaire/completed-open.html

```diff
@@ -1,11 +1,5 @@
 [% IF c.cobrand.is_council %]
-<p style="font-size: 150%">
-Thank you very much for filling in our questionnaire.
-</p>
+    <p>Thank you very much for filling in our questionnaire.</p>
```

Comment:
The moving of the h1 outside the `if` in the completed template means it will now say thanks twice: "Thank you for your feedback / Thank you very much for filling in our questionnaire" - is that intended?